x/stakeibc/keeper/trade_route_test.go

```diff
@@ -0,0 +1,119 @@
+package keeper_test
+
+import (
+	"strconv"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	keepertest "github.com/Stride-Labs/stride/v14/testutil/keeper"
+	"github.com/Stride-Labs/stride/v14/testutil/nullify"
+	"github.com/Stride-Labs/stride/v14/x/stakeibc/keeper"
+	"github.com/Stride-Labs/stride/v14/x/stakeibc/types"
+)
+
+func createNTradeRoute(keeper *keeper.Keeper, ctx sdk.Context, n int) []types.TradeRoute {
+	items := make([]types.TradeRoute, n)
+	for i := range items {
+
+		hostChain := strconv.Itoa(i) + "chain"
+		rewardChain := strconv.Itoa(i+1) + "chain"
+		tradeChain := strconv.Itoa(i+2) + "chain"
+
+		hostICA := types.ICAAccount{
+			ChainId: hostChain,
+			Type: types.ICAAccountType_WITHDRAWAL,
+		}
+		rewardICA := types.ICAAccount{
+			ChainId: rewardChain,
+			Type: types.ICAAccountType_UNWIND,
```

Comment:
do we want to keep this as "UNWIND"?

Reply:
You could change it to REWARD if you want, this is the main one which really doesn't matter because pfm means we never directly use this to send messages from Stride chain so the ICAAccountType is never used to figure out the owner, portID, etc